global_planner/include/global_planner/planner_core.h

```diff
@@ -193,7 +193,7 @@ class GlobalPlanner : public nav_core::BaseGlobalPlanner {
         int publish_scale_;
 
         void outlineMap(unsigned char* costarr, int nx, int ny, unsigned char value);
-        unsigned char* cost_array_;
```

Comment:
This is unused in the package.

Reply:
This seems like an unnecessary ABI break.